### ISSUE TITLE
Accessibility: Keyboard users can select and move cels

### ISSUE THREAD

**Issue by emhoracek** [emhoracek/smooch#29]
Right now the canvas isn't accessible for keyboard users. 

It would be cool if you could somehow select into the canvas, maybe using TAB, and then it would select a cel. The selection could be shown by adding a "halo" effect by lightening and blurring a copy of the cel image behind the cel itself. The user should be able to move the selected cel by moving the arrow keys. They could change which cel is selected by hitting TAB and cycle through cels that way.

If this seems like too big a task, feel free to take just part of the task. For example, you could just do the selection visualization and leave moving the cel with arrow keys to someone else.
